src/main/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistant.java

```diff
@@ -111,15 +111,21 @@ public void editAssignmentInCanvas(Long courseId, AttendanceAssignment attendanc
     }
 
     public void deleteAssignmentInCanvas(Long canvasCourseId, OauthToken oauthToken) throws AttendanceAssignmentException{
-
         List<AttendanceSection> sections = attendanceSectionService.getSectionByCanvasCourseId(canvasCourseId);
         if(CollectionUtils.isEmpty(sections)) {
-            throw new AttendanceAssignmentException(AttendanceAssignmentException.Error.NON_EXISTENT_SECTION_ERROR);
+            LOG.warn("Sections empty");
```

Comment:
If you're going to log this at this high of a logging level, (which you probably should) then you need to include identifying information in the message, such as the canvas course Id. Otherwise it's worthless when parsing production logs. Anything logged above Debug level will be in production logs, and so it needs identifying information to tell who it's talking about.